easybuild/easyconfigs/s/SVG/SVG-2.87-foss-2022a.eb

```
@@ -0,0 +1,26 @@
+easyblock = 'PerlModule'
+
+name = 'SVG'
+version = '2.87'
+
+homepage = 'https://metacpan.org/pod/SVG'
+description = "Perl binding for SVG"
+
+toolchain = {'name': 'foss', 'version': '2022a'}
```

I would think that `GCCcore` would be sufficient?
```suggestion
toolchain = {'name': 'GCCcore', 'version': '11.3.0'}
```

---

Ah, yes, the previous version was in foss. Resolved in https://github.com/easybuilders/easybuild-easyconfigs/pull/17212/commits/ab01bd22c6e71b2c9027ca807c8e3fdb859bb240